test/Microsoft.ApplicationInsights.AspNet.Tests/Helpers/HttpContextAccessorHelper.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Microsoft.ApplicationInsights.DataContracts;
+using Microsoft.AspNet.Hosting;
+using Microsoft.AspNet.Http;
+using Microsoft.AspNet.Http.Core;
+using Microsoft.AspNet.Mvc;
+using Microsoft.Framework.DependencyInjection;
+using System;
```

Comment:
Since we are staying with the Core coding style, using directives need to be sorted system-first and placed inside of the namespace.